### PR TITLE
Make BatchedDot don't try to use c code if there is no blas.

### PR DESCRIPTION
**Pull request by nouiz**
This fix crash in travis. I don't know why the new travis don't link anymore to blas. But this fix a bug in that case.

### CODE REVIEW

**Comment by lamblin** [comment]
I thought we managed to link with blas on Travis.
Did something change? @abergeron

File: theano/tensor/blas.py [hunk 1]
```diff
@@ -2134,6 +2134,10 @@ def c_code(self, node, name, inp, out, sub):
         _z, = out
         fail = sub["fail"]
 
+        if not config.blas.ldflags:
+            return super(BatchedDot, self).c_code(node, name,
+                                                  inp, out, sub)
```
Comment (lamblin) [inline theano/tensor/blas.py]: Is it possible for the error message to say that BLAS is needed?

**Comment by lamblin** [comment]
I'm OK with skipping the tests, but we would need something to tell us when linking with BLAS on Travis is broken, so we can fix it.

File: theano/tensor/blas.py (hunk 1 above)
Comment (nouiz) [inline theano/tensor/blas.py]: This don't raise an error. It just skip the c code for this op. It have a perform, so it won't cause crash later.

**Comment by nouiz** [comment]
Agreed, but I let this for another PR to get this one merged sooner.

We can add an assert that blas.ldflags isn't empty.

**Comment by abergeron** [comment]
What changed is what conda installs.  We no longer install the mkl version of numpy and we no longer install the mkl-rt pacakge.  I guess this is something that changed with a very recent update of conda itself.

**Pull request merged**